employees_info.py

```diff
@@ -0,0 +1,60 @@
+
+import tables
+import options
+
+
+create_employees_info = """create table employees_info(
+    name text not null,
+    country_code text,
+    salary real not null);"""
+
+create_countries = """create table countries (
+        code text not null,
+        country text not null);"""
+
+
+def main():
+    connection = tables.create_connection('employees.db')
+
+    if tables.check_if_table_exists(connection, 'employees_info') is False:
```

Comment:
maybe it makes sense to move this code to another place that is more related to working with tables?